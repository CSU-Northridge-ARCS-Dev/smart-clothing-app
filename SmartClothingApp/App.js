import { StatusBar } from 'expo-status-bar';
import React from 'react';
import { StyleSheet, Text, View, Image, SafeAreaView } from 'react-native';

// Background Image placeholder
const PlaceholderImage = require('./assets/images/background-image.png')

export default function App() {
  return (
    <SafeAreaView style={styles.container}>
      {/* Text content */}
      <Text>Welcome to the Smart Clothing Application!</Text>
      <StatusBar style="auto" />

      {/* Image container */}
      <View style={styles.imageContainer}>
        {/* Background Image */}
        <Image source={PlaceholderImage} style={styles.image} />
      </View>
    </SafeAreaView>
  );
}

{/* Background Image */}
const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
  imageContainer: {
    marginTop: 20,
    alignItems: 'center',
  },
  image: {
    width: 320,
    height: 440,
    borderRadius: 18,
  },
});
