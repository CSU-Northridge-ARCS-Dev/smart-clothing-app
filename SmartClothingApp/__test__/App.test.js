import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

describe('App component <App />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<App />).toJSON(); 
        expect(tree).toMatchSnapshot();
    });
});