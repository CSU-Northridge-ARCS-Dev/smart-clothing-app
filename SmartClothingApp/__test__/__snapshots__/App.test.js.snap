// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component <App /> renders correctly 1`] = `null`;
